internal/compiler-bridge-test/src/test/scala/xsbt/ExtractUsedNamesNamingConventionSpecification.scala

```diff
@@ -0,0 +1,306 @@
+package xsbt
+
+import sbt.internal.inc.UnitSpec
+
+class ExtractUsedNamesNamingConventionSpecification extends UnitSpec {
+
+  "Variable declarations and definitions" should "represented by multiple symbols" in {
+    //-----------0---------1---------2---------3---------4
+    //-----------01234567890123456789012345678901234567890
+    val src = """abstract class C(val xp: Int) {
+                |  val xm: Int = ???
+                |  val xam: Int
+                |  private[this] val xlm: Int = ???
+                |  def m = {
+                |    val xl: Int = ???
+                |    type S = { val xs: Int }
+                |    type E = xe.type forSome { val xe: AnyRef }
+                |  }
+                |}""".stripMargin
+    val compilerForTesting = new ScalaCompilerForUnitTesting
+    val (_, referencedSymbols, definedSymbols) =
+      compilerForTesting.extractUsedNamesFromSrc(src)
+    val expectedReferencedSymbols = Set(
+      (0, 25, "scala.Int#"),
+      (1, 10, "scala.Int#"),
+      (1, 16, "scala.Predef#$qmark$qmark$qmark()."),
```

Comment:
Symbolic identifiers should be in decoded form instead of encoded as here. This particularity is not documented in the spec, I will update it https://github.com/scalameta/scalameta/issues/1510

Reply:
OK. Thank you.